tests/baselines/reference/globalThisCollision.errors.txt

```diff
@@ -0,0 +1,7 @@
+tests/cases/conformance/es2019/globalThisCollision.js(1,5): error TS2397: Declaration name conflicts with built-in global identifier 'globalThis'.
+
+
+==== tests/cases/conformance/es2019/globalThisCollision.js (1 errors) ====
+    var globalThis;

```

Comment:
nitpit: an even better test would be `var globalThis = window` or `var globalThis = this` since that's the likely polyfill code we'll see in JS.

Reply:
I don't think its necessary since that doesn't effect the outcome of the test, this is purely testing that `var globalThis` in the global scope results in our collision error message.